static/flow/views/program-editor-panel.js

```diff
@@ -330,6 +330,7 @@ var ProgramEditorPanel = function(options) {
                                 inputephemeral.mousedown(function(e) {e.stopPropagation()});
                                 var eventdataephemeral = {blockid:block.id, paramname: "sequence_names", connectedblockid:connectedblockid, divindex:divindexephemeral };
                                 inputephemeral.keyup(eventdataephemeral, _this.paramEntryChanged);
+								inputephemeral.focusout(eventdata, _this.paramEntryFocusOut);
```

Comment:
Not critical but this is mixing tabs and spaces.

Reply:
Just pushed a few more changes to unify whitespace. I also had a couple of obsolete functions still hanging around that I had moved into Utils.

Reply:
I've been trying to unify tabs/spacing as I progress through modules.  There was a mixture of styles before.

Reply:
Eventually we ideally should go through all modules and unify the whitespace style.  Might be worth putting as a tech debt story in Pivotal Tracker.